Sources/SotoS3FileTransfer/S3FileTransferManager.swift

```diff
@@ -265,7 +265,8 @@ public struct S3FileTransferManager {
         return listFiles(in: folder)
             .flatMap { files in
                 let taskQueue = TaskQueue<Void>(maxConcurrentTasks: configuration.maxConcurrentTasks, on: eventLoop)
-                let transfers = Self.targetFiles(files: files, from: folder, to: s3Folder)
+                let folderResolved = URL(fileURLWithPath: folder).standardizedFileURL.resolvingSymlinksInPath()
```

Comment:
Any reason you don't put this inside of `targetFiles`

Reply:
Yes I had this inside of `targetFiles` initially, but then I read the comments for all 3 `targetFiles` methods that say `Function assumes the files have srcFolder prefixed`. So I didn't want to change the assumption for one of them.  But I can move `let folderResolved = URL...` inside `targetFiles` if you wish.